### PR TITLE
Added a release process for ETOS.

### PR DESCRIPTION
**Pull request by t-persson**
### Applicable Issues

None

### Description of the Change

Add a release process to explain how ETOS is released and what to expect from us.

### Alternate Designs

There are multiple designed, but during discussion with maintainers this was the proposed one.

### Benefits

Easier for us to keep track and users will be happy knowing what happens in the world of ETOS.

### Possible Drawbacks

We must stick to it :)

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Tobias Persson <tobias.persson@axis.com>

### CODE REVIEW

File: source/release_process.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+The ETOS workflow is to work with milestones.
+
+Each milestone is considered a new release with a version number (:ref:`either minor or major <versioning>`) and the issues that are solved in milestones will be added to that release.
+(Note that we might hold back issues for a release if they are not deemed stable enough, but we will communicate this by removing the issue from the milestone).
+
+These miletones are updated and created every friday and there will always be at least two future milestones in the works so that users have a heads-up what will be coming in the future.
+
+If backwards compatibility is broken then the feature will be deprecated, the new one activated and an issue for removing it will be added to a major milestone in the future so that users know ahead of time whene a breaking change is coming.
+
+This means that, at worst, users get a 4 week heads up for breaking changes.
```
Comment (andjoe-axis) [inline source/release_process.rst:21]: Possibly we can add a clarification eg:

"This means that, at worst, users get a 4 week heads up for breaking changes. And by breaking changes, we mean that a developer may have to change something in their code before they can upgrade."

**Review by andjoe-axis — APPROVED** [review]
+1

**Review by fredjn — CHANGES_REQUESTED** [review]
We should add a link to one of our github milstones as an example

**Review by fredjn — APPROVED** [review]
+1

**Pull request merged**